release-notes/con-relnotes-notable-features.adoc

```diff
@@ -3,143 +3,129 @@
 
 This section highlights new features in {product} {product-version}.
 
-== Support for Role-Based Access Control 
 
-The Role-Based Access Control (RBAC) feature in {product} introduces an administrator role and optimizes access control through organizational structures such as teams, groups, and users.
+== Support for upgrading the {product} Helm Chart 
 
```

Comment:
Instead of the Helm Chart information I would mention that RHDH is now based on the Backstage v1.23.4. that that is more important information

Reply:
== Change in productized version  

{product} is now a fully supported, enterprise-grade productized version of upstream Backstage v1.23.4.